x-pack/test/api_integration/apis/metrics_ui/log_threshold_alert.ts

```diff
@@ -0,0 +1,236 @@
+/*
```

Comment:
One more question about this file: Do we want to put it into the `logs_ui` directory since it covers a log rule?

Reply:
Makes sense... moving it there.